website/content/sdk/guides/intros/stocks/stocks-screener.md

```diff
@@ -0,0 +1,314 @@
+
+Price/Free Cash Flow = Any
+```
+
+## Examples
+
+To make a screener with only a few filters, it should look something like this:
+
+```console
+# Author of preset: Danglewood
+# Description: SDK Demo Screener
+
+[General]
+Order = Relative Volume
+Signal = Major News
+
+[Descriptive]
+Average Volume = Over 500K
+Price = Over $5
+
+[Fundamental]
+Price/Free Cash Flow = Low (<15)
+
+[Technical]
+Beta = Under 1
+```
+
+Copy the block above to a new text file in any editor, and save the file to the OpenBBUserData folder, naming it something like, `sdk_guide_preset.ini`. Declaring the path to the preset file, when located in the OpenBBUserData folder, is not required. The kernel must be restarted in order for the file to be recalled this way; however, changes to the preset itself will be reflected without a restart. 
+
+**This preset file has been included in the ZIP file at the top of the page.**
+
+It is a good idea to test choices made before making it too complicated. Start with a handful of filters and modify, or add, them one-at-a-time. Let's pass what we have so far through the screener with `overview` selected as the `data_type`.
+
+### Performance
+
+```python
+openbb.stocks.screener.screener_data(preset_loaded='sdk_guide_preset.ini', data_type = 'performance', limit = 100)
+```
+
+|    | Ticker   |   Perf Week |   Perf Month |   Perf Quart |   Perf Half |   Perf Year |   Perf YTD |   Volatility W |   Volatility M |   Recom |    Avg Volume |   Rel Volume |   Price |   Change |           Volume |
+|---:|:---------|------------:|-------------:|-------------:|------------:|------------:|-----------:|---------------:|---------------:|--------:|--------------:|-------------:|--------:|---------:|-----------------:|
+|  0 | GEO      |      0.1389 |       0.4005 |       0.4589 |      0.6723 |      0.3955 |     0.5342 |         0.0574 |         0.0558 |     2.5 |      2.09e+06 |         4.23 |   11.89 |   0.1623 |      8.86013e+06 |
+|  1 | EBS      |     -0.0059 |      -0.4513 |      -0.5136 |     -0.6429 |     -0.7204 |    -0.7292 |         0.0452 |         0.0827 |     2.7 | 753610        |         2.91 |   11.77 |   0.0138 |      2.19107e+06 |
+|  2 | TME      |      0.1871 |       0.7984 |       0.3721 |      0.5789 |     -0.0531 |    -0.0365 |         0.0681 |         0.0756 |     2.3 |      9.95e+06 |         2.2  |    6.6  |   0.0611 |      2.18486e+07 |
+|  3 | QFIN     |      0.0874 |       0.4331 |      -0.0738 |     -0.0744 |     -0.389  |    -0.365  |         0.0472 |         0.0787 |     1.4 | 997070        |         1.62 |   14.56 |   0.0849 |      1.61386e+06 |
+|  4 | ATHM     |      0.0229 |       0.0391 |      -0.1772 |     -0.2256 |     -0.1971 |    -0.0455 |         0.0492 |         0.0579 |     2.2 | 577050        |         1.6  |   28.14 |   0.0484 | 922812           |
+...continued...
```

Comment:
Please remove this continued statements and either do `...` or put continued at the bottom. It messes up the table.

Reply:
Removed the continued as the index numbers should provide some context.